src/entities/purchase-params.ts

```diff
@@ -21,4 +22,26 @@ export interface PurchaseParams {
    * The email of the user. If undefined, RevenueCat will ask the customer for their email.
    */
   customerEmail?: string;
+
+  /**
+   * Optional customization for the translations used in the purchase flow.
+   * The keys are the locale codes and the values are objects with the same keys as the default translations.
+   */
+  customTranslations?: CustomTranslations;
```

Comment:
TBH I'm a bit reticent to provide this API unless it's something requested often by developers... I think I would prefer to have more control over the text and lower the constraints in the future if it's really needed. Wdyt?

Reply:
Right... If we end up moving to using enums, it would mean that we might want to make the enum public, and if we want to change that enum for any reason, it would be a breaking change... I guess we can keep this interface using strings, but as I mentioned, I think that's error prone... I guess we can keep it for now even if not the easiest of interfaces, but as mentioned in a different comment, I would suggest moving it to configure time to clear the parameters for the actual purchase/renderPaywall methods.

Reply:
I already migrated all the labels to be an enum!

Re: custom overrides, I think it's fine, worst case they will be ignored (if a key is wrong or renamed).

We should consider changes to the labels breaking anyway so better to know exactly which ones they are with an enum.

Reply:
I like that, I'll give it a go tomorrow since I am also renaming the labelId.

Thanks for all the feedback @tonidero!